views/userRankPage.hbs

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>DeadLock - Ask A Question</title>
+    <meta charset="utf-8">    
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- External JavaScript -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <script src="javascripts/question-prompt.js"></script>
+
+    <!-- External CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <link rel="stylesheet" href="stylesheets/standardstyle.css">
+    <link rel="stylesheet" href="stylesheets/userrankpagestlye.css">
+   
+  </head>
+  <body>
+    {{>header}}
+      <!-- Jumbotron containing all tags -->
+      <div class="jumbotron container-fluid text-center" style="margin-top: 25px">
+        <h2>Select a tag</h2>
+        <p>View all users based on tag score</p>
+        <div class="btn-group-wrap">
+          <div class="btn-group">
+              <button type="button" class="btn btn-success">Java</button>
```

Comment:
Order of properties does not match code styling guide - please put `class` before all other properties. Please do this for all buttons below as well.